holidays/countries/bermuda.py

```
@@ -0,0 +1,141 @@
+
+        # Remembrance Day.
+        self._add_observed(self._add_remembrance_day("Remembrance Day"))
+
+        # Christmas Day.
+        self._add_observed(self._add_christmas_day("Christmas Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)
+
+        # Boxing Day.
+        self._add_observed(self._add_christmas_day_two("Boxing Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)
+
+
+class BM(Bermuda):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)